core/client/src/main/java/alluxio/client/file/options/SetAttributeOptions.java

```diff
@@ -123,7 +123,6 @@ public boolean hasOwner() {
    * @return the owner
    */
   public String getOwner() {
-    Preconditions.checkState(hasOwner(), PreconditionMessage.MUST_SET_OWNER);
```

Comment:
the SetAttribute options class is the only class when null makes sense as the default value, for other options we actually want to provide defaults ... so I don't think this change needs to be propagated